packages/ui/src/council/components/election/pastElection/PastElectionTabs.tsx

```diff
@@ -15,13 +16,22 @@ const getMyVote = (votingResult: ElectionVotingResult, myAccounts: Account[]) =>
   return votingResult.votes.find((vote) => myAccounts.find((account) => account.address === vote.castBy))
 }
 
+export const electionVotingResultComparator = <T extends { totalStake: BN; votesNumber: number }>(
+  votingResult1: T,
+  votingResult2: T
+) => {
+  const stakeOutcome = Comparator<T>(true, 'totalStake').bigNumber(votingResult1, votingResult2)
+  return stakeOutcome === 0 ? applyOrder(votingResult1.votesNumber - votingResult2.votesNumber, true) : stakeOutcome
+}
```

Comment:
Please could this be in `@/council/model/electionVotingResultComparator.ts` ?
Also this is a bit clearer to me:
```suggestion
) =>
  Comparator<T>(true, 'totalStake').bigNumber(votingResult1, votingResult2) ||
  Comparator<T>(true, 'votesNumber').number(votingResult1, votingResult2)
```